### PR TITLE
Tech Yes We Can PT 1 

### PR DESCRIPTION
**Pull request by ritabarlow**
Part one of a two part blog series from the six ladies that attended the women in tech conference.

### CODE REVIEW

File: posts/TechYesWeCan_Pt1.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+---
+authors:
+- Leigha Dellinger, Sarah Narum, Rita Barlow, Trish Brumett, Erin Geoghan, Ramya Shetty
```
Comment (aseigneurin) [inline posts/TechYesWeCan_Pt1.md]: Should be one author per line

File: posts/TechYesWeCan_Pt1.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+tags:
+- Women In Technology
+- RVATech
+date: 2018-10-31T12:21:50.000Z
+title: "Tech Yes We Can! Version 1.1"
+image: 
+---
+
+# 'Tech Yes She Can!’ Version 1.1
```
Comment (aseigneurin) [inline posts/TechYesWeCan_Pt1.md]: Remove this title, this is redundant with the title in the meta information

File: posts/TechYesWeCan_Pt1.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+It was advised, and properly so, that the ramblings of six women and their experience at a conference may be a bit overwhelming to society, so we broke this inspirational knowledge fest into a two-parter. We hope that you find something here to inspire you, regardless of gender, to go out into the world and make a difference, even if that difference is solely introspective. We have done our best to carry on the essence of the conference within our tales and more importantly bring along our own experiences in order to offer some sort of guidance if you are here to seek it. Although we do not speak for RVATech or all women, we do speak as six women IN technology from various backgrounds, points in our careers, and stylistic preferences. Take what you want, leave what you don’t and comments or feedback are encouraged!
+
+
+>## Sarah Narum
```
Comment (aseigneurin) [inline posts/TechYesWeCan_Pt1.md]: `>` is for a block quote - would be better to use a proper title

File: posts/TechYesWeCan_Pt1.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+The concept of inclusion was highly talked about, which most often refers to inclusion within a network of people but I believe it applies within oneself as well. I come from a diverse family with my artist father and my programmer mother, so when Avis Yates Rivers said “bring your whole self to whatever you do”, I could not relate more. I’ve found that many times our strengths aren’t always the traditional skill set associated with a job or task. I believe the artist in me allows me to problem solve differently than others on my team and my overall confidence in my creative skills drives me to improve my technical skills. As an art student you put your art, and symbolically yourself, on a pedestal for critiquing which forces you to face your fears of failing. Once again as Rivers stated, you must leave your fears of how you will be interpreted and ‘bring yourself anyways’.
```
Comment (aseigneurin) [inline posts/TechYesWeCan_Pt1.md]: Missing blank line between these 2 paragraphs

File: posts/TechYesWeCan_Pt1.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+**Creative Spirit**
+
+With my background it will be no shock that I wholeheartedly believe in the importance
+of one’s creative spirit. Many people believe creativity is only linked to being artistic or an artist but I’ve never believed these to be mutually inclusive. To me, creativity is about passion, innovation and curiosity, no matter the subject matter or skill. During Rivers’ segment on ‘Necessary Inclusion’, she speaks of the importance of individual experiences within innovation. Multiple people can be given the same task, product, or situation and each will execute, experience and react differently based on their personality and whole life experiences. This is how we all can continue to progress, especially in the world of technology. In conclusion of the conference, MC Ferrell made a vibrant declaration that ‘we have to be intentional about getting inspired’ and we do! Sometimes that means taking a break from the old and trying something new or maybe returning to the familiar to boost our spirits before conquering a puzzling dilemma at work or home.
```
Comment (aseigneurin) [inline posts/TechYesWeCan_Pt1.md]: Remove the line return here

File: posts/TechYesWeCan_Pt1.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+
+
+>## Sarah Narum
+>Ippon Engineer | Data | snarum@ipponusa.com
+
+This was my second year attending the RVATech: Women in Technology conference and once again I was blown away by the vivacious, empowering women who led (and attended) this conference. The ever-so-energetic MC, Courtney Ferrell, summarized it best when she said, ‘you need three things to be successful: Confidence, Creative Spirit and Connections’. All of the speakers touched on the importance of these three factors of success in their own way so I’ve decided to address each:
```
Comment (aseigneurin) [inline posts/TechYesWeCan_Pt1.md]: Use double quotes to quote text throughout the post

File: posts/TechYesWeCan_Pt1.md [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+
+**Connections**
+
+The final element for success is connections. This is what I personally struggle with most, at least when I have to be intentional about it. I am a bubbly person socially but the world of networking is not a comfort zone for me and I know it is not for many. Some of the best advice I took from this conference is about making new connections. During a break-out session about ‘Building Your Personal Brand’, one speaker said ‘you must cast a wide net’ to capture people of interest and from there, you can start narrowing down your network. If you start out too exclusive in your search for connections, you are more likely to block yourself from opportunities. Again, this is easier said than done (for me at least). It takes a lot of courage and effort, but the number one part to nurturing your connections is communication. I cannot stress this enough - we are beings dependant on communication whether its direct or inferred through verbal language, body language, social cues, etc. I think we all have to take care and time in communicating clearly, intentionally and often, and take into consideration that not everyone communicates in the same way as you.
```
Comment (aseigneurin) [inline posts/TechYesWeCan_Pt1.md]: its -> it's

File: posts/TechYesWeCan_Pt1.md [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+
+The message; Your personal brand paves the way for the impact you leave behind, every post, every comment, every action determines your professional and personal footprint and each should be calculated and methodical in order to be successful. Right? Sounds perfect, let’s get started...but how? What does the first step look like? What happens when you hit that inevitable setback or encounter that first devastating failure? How do you handle all the criticism and feedback and continue to show up each day putting your best foot forward? Better yet, if you are mid-career how do you morph an existing personal brand that you may have outgrown into the career person you are today?
```
Comment (aseigneurin) [inline posts/TechYesWeCan_Pt1.md]: `...but` -> `... but`

File: posts/TechYesWeCan_Pt1.md [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+
+I can’t answer all these questions simply from attending the Women in Tech conference but I can tell you that I listened to a slew of speakers, all women, all successful.  Not a single one of them had an easy road, none of them fell into success or got where they are without dusting a little dirt off and rising to the occasion. Each speaker told a different story but they all carried a similar theme: Be confident, know your worth, bring your whole self to the table, be utterly relentless in the pursuit of your passion, and always live your purpose.
```
Comment (aseigneurin) [inline posts/TechYesWeCan_Pt1.md]: `theme: Be ` -> `theme: be `

File: posts/TechYesWeCan_Pt1.md [hunk 10]
```diff
@@ -0,0 +1,115 @@
...
+
+I can’t answer all these questions simply from attending the Women in Tech conference but I can tell you that I listened to a slew of speakers, all women, all successful.  Not a single one of them had an easy road, none of them fell into success or got where they are without dusting a little dirt off and rising to the occasion. Each speaker told a different story but they all carried a similar theme: Be confident, know your worth, bring your whole self to the table, be utterly relentless in the pursuit of your passion, and always live your purpose. 
+
+I won’t sugar coat this, that last one...living my purpose. That has been stuck with me since I walked out of that beautiful, yet extremely not conference friendly, train station and went on about my life. One word, purpose. What is my purpose?
```
Comment (aseigneurin) [inline posts/TechYesWeCan_Pt1.md]: `one...living` -> `one... living `

File: posts/TechYesWeCan_Pt1.md [hunk 11]
```diff
@@ -0,0 +1,115 @@
...
+
+I still don’t have the answer and it’s been a week, I know….an entire week and I haven’t solved this tiny little problem, ridiculous! I think what’s more interesting is that after spending many dollars, eating too many boxed lunches, sitting in too many uncomfortable chairs, listening to people talk at me year after year at conferences I am still thinking about what was said after a week. For a skeptic conference attendee like me - that is a first.
```
Comment (aseigneurin) [inline posts/TechYesWeCan_Pt1.md]: `know….an` -> `know... an `

File: posts/TechYesWeCan_Pt1.md [hunk 12]
```diff
@@ -0,0 +1,115 @@
...
+
+If you have not yet attended this conference, do yourself a favor and add it to your list for next year.  Not only is it amazing to see so many women from all over the world come together in support of technology it is sobering to hear that despite all our success and triumphs we all still struggle with the fundamentals of being human. 
+
+>## Leigha Dellinger
+>Ippon Engineer & Tech Manager | Front End | ldellinger@ipponusa.com
+
+Looking at the lineup prior to buying tickets for this years conference, I was a little nervous. These speakers didn’t sound like they would be as IT-focused as those I’d heard from in the past. I was worried that after seven years of attendance I wouldn’t be able to say rvatech/women had yet again outdone themselves. Boy was I wrong!
```
Comment (aseigneurin) [inline posts/TechYesWeCan_Pt1.md]: `years` -> `year's`

File: posts/TechYesWeCan_Pt1.md [hunk 13]
```diff
@@ -0,0 +1,115 @@
...
+>## Leigha Dellinger
+>Ippon Engineer & Tech Manager | Front End | ldellinger@ipponusa.com
+
+Looking at the lineup prior to buying tickets for this years conference, I was a little nervous. These speakers didn’t sound like they would be as IT-focused as those I’d heard from in the past. I was worried that after seven years of attendance I wouldn’t be able to say rvatech/women had yet again outdone themselves. Boy was I wrong! 
+
+We heard from astounding women with unique perspectives on how things are changing and adapting in the technology industry for not only gender differences but for diversity in general.
+
+Bringing Courtney Ferrell back again this year was the first step towards an amazing conference. She has such passion for everything she says. Early morning conferences can start at a bit of a drag but it was hard to even show an ounce of sleepiness with Courtney up on stage - brightening anything and everything in the room. “Be intentional about being inspired” was one of many favorite phrases from the day. 
+
+**Kate Foster | Principal | KickStart Specialists**
```
Comment (aseigneurin) [inline posts/TechYesWeCan_Pt1.md]: Use a title instead

File: posts/TechYesWeCan_Pt1.md [hunk 14]
```diff
@@ -0,0 +1,115 @@
...
+
+I have said in the past that getting a degree in Business Information Systems made a world of difference in my career. Having that engineering background with a mix of business concepts and understanding has helped me maintain relationships with business leaders within every project I have been a part of. Hearing Kate Foster speak towards just this was perfect!
+
+IT and business people don’t speak the same language, everyone knows that. But what few engineers learn early on is that you must be able to translate for the business. IT “gobbledygook” causes problems -- not matter how good your solution is.
```
Comment (aseigneurin) [inline posts/TechYesWeCan_Pt1.md]: `not matter` -> `no matter`

File: posts/TechYesWeCan_Pt1.md [hunk 15]
```diff
@@ -0,0 +1,115 @@
...
+
+**Kate Foster | Principal | KickStart Specialists**
+
+I have said in the past that getting a degree in Business Information Systems made a world of difference in my career. Having that engineering background with a mix of business concepts and understanding has helped me maintain relationships with business leaders within every project I have been a part of. Hearing Kate Foster speak towards just this was perfect!
+
+IT and business people don’t speak the same language, everyone knows that. But what few engineers learn early on is that you must be able to translate for the business. IT “gobbledygook” causes problems -- not matter how good your solution is. 
+
+What good are innovative ideals if you can’t explain why they’re good. Ineffective communication is one of the primary contributions to project failures ⅓ of the time (and has a negative effect ½ of the time). 
+And even if you know how to communicate, how do you know the message you’re sending is what they’re receiving? Get to know the client! And get to know yourself.
```
Comment (aseigneurin) [inline posts/TechYesWeCan_Pt1.md]: Either remove the line return or add one between these 2 lines

File: posts/TechYesWeCan_Pt1.md [hunk 16]
```diff
@@ -0,0 +1,115 @@
...
+IT and business people don’t speak the same language, everyone knows that. But what few engineers learn early on is that you must be able to translate for the business. IT “gobbledygook” causes problems -- not matter how good your solution is. 
+
+What good are innovative ideals if you can’t explain why they’re good. Ineffective communication is one of the primary contributions to project failures ⅓ of the time (and has a negative effect ½ of the time). 
+And even if you know how to communicate, how do you know the message you’re sending is what they’re receiving? Get to know the client! And get to know yourself. 
+
+**Avis Yates Rivers | CEO | Technology Concepts Group International**
+Women hold a meaningful percentage of participation in the technology world. We comprise 57% of US professional occupations, 26% of US tech jobs and 19% of the US software developers. These numbers sound about right! We are growing in this industry and I feel a shift in the balances. But even with all that change and time we are still only hold 5% of technology leadership jobs. Avis Rivers spoke to this idea; that the gaps in this industry aren’t just about pay but also still with gender and race.
```
Comment (aseigneurin) [inline posts/TechYesWeCan_Pt1.md]: `we are still only hold` -> `we still only hold`?

File: posts/TechYesWeCan_Pt1.md [hunk 17]
```diff
@@ -0,0 +1,115 @@
...
+
+Being aware of the problem, she described best as intersectionality, we can better acknowledge and ground the differences among us. We can break out of these discriminations and disadvantages. 
+
+Step 1: “Bring your whole authentic self to work and what your doing”. We’re all different. Diversity of thought leads to better problem solving. But it’s this combination that drives innovation! Engage all the voices on your team. Don’t filter questions or solutions based on race or age. We need everyone represented in each business. Change the dynamic!
+
+Step 2: Understand that equality doesn’t work for everyone. Equity is what we should be aiming for to create success! Rivers showed an image, that many people probably recognize, during her talk. It resonated with me and really shows the difference in these two types of strategies vs the reality of life.
+
+<responsive-figure src="https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2018/10/Leigha_IMG1.png">
```
Comment (aseigneurin) [inline posts/TechYesWeCan_Pt1.md]: Use the `~[title](url)` syntax

File: posts/TechYesWeCan_Pt1.md [hunk 18]
```diff
@@ -0,0 +1,115 @@
...
+
+*Good solutions require insight and data into that problem.*
+
+Nancy then went on to describe how the world is changing and how building and designing for that world requires a different way of things. We as engineers need to be building for a more inclusive world. One fabulous example she gave was changing stations only existing in womens restrooms. Who thought that women would be the only ones out and about needing to change a baby's diaper? But then again -- why would it cross your mind as any other way? It’s the way it’s always been. We need to change our thoughts, test and design differently.
+ 
+
+As I’ve said in past articles, this conference only gets better year by year. They have an amazing group of women (and men) that take each attendees thoughts into account and put that into the following year’s conference. The speakers always blow me away, but I was struck extra hard this year; these keynotes really hit it home. They all had something totally different to say but with one major thought -- build yourself up! Be the best you that you can be by getting to know yourself better.
+
+
```
Comment (aseigneurin) [inline posts/TechYesWeCan_Pt1.md]: Use a proper separation (multiple empty lines all end up being just one)

File: posts/TechYesWeCan_Pt1.md [hunk 19]
```diff
@@ -0,0 +1,117 @@
+---
```
Comment (aseigneurin) [inline posts/TechYesWeCan_Pt1.md]: Re the file name: 

> Title: this should "match" the slug of the post, i.e. the name of the file after converting to lowercase and replacing spaces with dashes.

File: posts/TechYesWeCan_Pt1.md (hunk 1 above)
Comment (ritabarlow) [reply posts/TechYesWeCan_Pt1.md]: Noted and revised.

File: posts/TechYesWeCan_Pt1.md (hunk 2 above)
Comment (ritabarlow) [reply posts/TechYesWeCan_Pt1.md]: I thought it was nice formatting, however I have removed it!

File: posts/TechYesWeCan_Pt1.md (hunk 3 above)
Comment (ritabarlow) [reply posts/TechYesWeCan_Pt1.md]: Gotcha! Was just trying to break it up a bit so it was easier to read between the 3 authors.

File: posts/TechYesWeCan_Pt1.md (hunk 4 above)
Comment (ritabarlow) [reply posts/TechYesWeCan_Pt1.md]: Got it.

File: posts/TechYesWeCan_Pt1.md (hunk 5 above)
Comment (ritabarlow) [reply posts/TechYesWeCan_Pt1.md]: Got it

File: posts/TechYesWeCan_Pt1.md (hunk 6 above)
Comment (ritabarlow) [reply posts/TechYesWeCan_Pt1.md]: I copied and pasted from three authors so I'm hoping I caught them all!

File: posts/TechYesWeCan_Pt1.md (hunk 7 above)
Comment (ritabarlow) [reply posts/TechYesWeCan_Pt1.md]: corrected

File: posts/TechYesWeCan_Pt1.md (hunk 8 above)
Comment (ritabarlow) [reply posts/TechYesWeCan_Pt1.md]: Revised

File: posts/TechYesWeCan_Pt1.md (hunk 9 above)
Comment (ritabarlow) [reply posts/TechYesWeCan_Pt1.md]: Revised

File: posts/TechYesWeCan_Pt1.md (hunk 10 above)
Comment (ritabarlow) [reply posts/TechYesWeCan_Pt1.md]: Revised

File: posts/TechYesWeCan_Pt1.md (hunk 11 above)
Comment (ritabarlow) [reply posts/TechYesWeCan_Pt1.md]: Revised

File: posts/TechYesWeCan_Pt1.md (hunk 12 above)
Comment (ritabarlow) [reply posts/TechYesWeCan_Pt1.md]: Revised

File: posts/TechYesWeCan_Pt1.md (hunk 18 above)
Comment (ritabarlow) [reply posts/TechYesWeCan_Pt1.md]: Got it

File: posts/TechYesWeCan_Pt1.md (hunk 19 above)
Comment (ritabarlow) [reply posts/TechYesWeCan_Pt1.md]: noted.

File: posts/TechYesWeCan_Pt1.md (hunk 17 above)
Comment (ritabarlow) [reply posts/TechYesWeCan_Pt1.md]: Um....I think I did this - I may need help!

**Comment by ritabarlow** [comment]
Help!

**Pull request merged**